extern/storage-sealing/precommit_batch_test.go

```diff
@@ -54,10 +54,11 @@ func TestPrecommitBatcher(t *testing.T) {
 			WaitDealsDelay:            time.Hour * 6,
 			AlwaysKeepUnsealedCopy:    true,
 
-			BatchPreCommits:     true,
-			MaxPreCommitBatch:   maxBatch,
-			PreCommitBatchWait:  24 * time.Hour,
-			PreCommitBatchSlack: 3 * time.Hour,
+			BatchPreCommits:            true,
+			MaxPreCommitBatch:          maxBatch,
+			PreCommitBatchWait:         24 * time.Hour,
```

Comment:
should we take this chance to lower it? SPs have been complaining this is too long